gateway/api/migrations/0033_job_trial_program_trial_instances_and_more.py

```diff
@@ -0,0 +1,33 @@
+# Generated by Django 4.2.16 on 2025-01-17 15:59
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ("auth", "0012_alter_user_first_name_max_length"),
+        ("api", "0032_computeresource_gpu_job_gpu"),
+    ]
+
+    operations = [
+        migrations.AddField(
+            model_name="job",
+            name="trial",
+            field=models.BooleanField(default=False),
+        ),
+        migrations.AddField(
+            model_name="program",
+            name="trial_instances",
+            field=models.ManyToManyField(
+                blank=True, related_name="program_trial_instances", to="auth.group"
+            ),
+        ),
+        migrations.AlterField(
```

Comment:
For who is interested on, this `AlterField` doesn't apply to the Database so we continue providing backward compatibility in the database in case we need to do rollback. It only affects to the ORM.